scripts/genpinctrl/genpinctrl.py

```diff
@@ -64,6 +64,7 @@
     "stm32l5": 0x42020000,
     "stm32mp1": 0x50002000,
     "stm32wb": 0x48000000,
+    "stm32wl": 0x48000000,
```

Comment:
This change should go into a separate commit. In order to make reviews easier, autogenerated content should go into separate commits.

Reply:
OK. Done